packages/orchestration/src/examples/evm-account-kit.js

```diff
@@ -0,0 +1,327 @@
+// @ts-check
+import { M, mustMatch } from '@endo/patterns';
+import { VowShape } from '@agoric/vow';
+import { makeTracer, NonNullish } from '@agoric/internal';
+import { atob, decodeBase64 } from '@endo/base64';
+import { defaultAbiCoder } from '@ethersproject/abi';
+import { Fail } from '@endo/errors';
+import { CosmosChainAddressShape } from '../typeGuards.js';
+import { buildGMPPayload } from '../utils/gmp.js';
+
+const trace = makeTracer('EvmTap');
+const { entries } = Object;
+
+const addresses = {
+  AXELAR_GMP:
+    'axelar1dv4u5k73pzqrxlzujxg3qp8kvc3pje7jtdvu72npnt5zhq05ejcsn5qme5',
+  AXELAR_GAS: 'axelar1zl3rxpp70lmte2xr6c4lgske2fyuj3hupcsvcd',
+};
+
+/**
+ * @import {IBCChannelID, VTransferIBCEvent} from '@agoric/vats';
+ * @import {Vow, VowTools} from '@agoric/vow';
+ * @import {Zone} from '@agoric/zone';
+ * @import {CosmosChainAddress, Denom, OrchestrationAccount} from '@agoric/orchestration';
+ * @import {FungibleTokenPacketData} from '@agoric/cosmic-proto/ibc/applications/transfer/v2/packet.js';
+ * @import {ZoeTools} from '@agoric/orchestration/src/utils/zoe-tools.js';
+ */
+
+/**
+ * @typedef {{
+ *   localAccount: ERef<OrchestrationAccount<{ chainId: 'agoric' }>>;
```

Comment:
```suggestion
 *   localAccount: HostInterface<OrchestrationAccount<{ chainId: 'agoric' }>>;
```

We don't need `ERef` since this object is in our contract vat. We also need to use `HostInterface` from `@agoric/async-flow` since `OrchestrationAccount` is typed for usage in async-flow functions. HostInterface will convert return signatures containing promises to vows and match what we'll actually have in this context.